deploy/onpublish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Travis build trigger should have environment variables COMMIT_USER, COMMIT_EMAIL and COMMIT_TOKEN setup.
+
+SOURCEFILENAME=".deploy/ddsclient-generic-gcb01.spec"
+TARGETFILENAME="ddsclient-$TRAVIS_TAG-gcb01.spec"
+COMMITMSG="DukeDSClient module update $TRAVIS_TAG"
+COMMITBRANCH="master"
+
+# Encode the spec file as base64
+BASE64CONTENT=`base64 $SOURCEFILENAME`
+
+# Upload file to helmod repo
+curl -i -X PUT -H "Authorization: token $COMMIT_TOKEN" \
```

Comment:
I'm thinking if a tag was deleted, moved to another commit, and re-pushed. I guess in practice, the content of the spec file wouldn't change, but the curl would probably return nonzero and cause the script to fail